src/components/layout/AddModal.jsx

```diff
@@ -46,8 +46,8 @@ const AddModal = props => {
 	return (
 		<span>
 			<Menu.Item key="4" className="menu-item" onClick={showModal}>
-				<span style={{ display: 'flex', alignItems: 'center' }}>
-					<FileAddFilled style = {{margin: '0 0.5rem 0 0'}}/>
+				<span style={{ display: 'flex', alignItems: 'center'}, props.style}>
```

Comment:
what is this added for?

Reply:
For the "Add Dashboard" on the sidebar I couldn't find where to change the font size in the SideBar.jsx so I passed in the font sizes based on winWidth as a prop, idk if there'd be a better way to do that 😓